CASMINST-7104 : passing image and config for rebuild (#5953)
Cray-HPE/docs-csm: operations/iuf/IUF.md

```text
usage: iuf run [-h] [-b BEGIN_STAGE] [-e END_STAGE] [-r RUN_STAGES [RUN_STAGES ...]] [-s SKIP_STAGES [SKIP_STAGES ...]] [-f]
               [-bc BOOTPREP_CONFIG_MANAGED] [-bm BOOTPREP_CONFIG_MANAGEMENT] [-bpcd BOOTPREP_CONFIG_DIR] [-rv RECIPE_VARS]

>                [-smc SET_MANAGEMENT_CONFIG] [-smi SET_MANAGEMENT_IMAGE]

Reviewer: Why did this get removed??
```
               [-sv SITE_VARS] [-mrs {reboot,stage}] [-cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE]
```

Author: added it . got removed by mistake

Reviewer: Okay, thank you.

```text
usage: iuf run [-h] [-b BEGIN_STAGE] [-e END_STAGE] [-r RUN_STAGES [RUN_STAGES ...]] [-s SKIP_STAGES [SKIP_STAGES ...]] [-f]
               [-bc BOOTPREP_CONFIG_MANAGED] [-bm BOOTPREP_CONFIG_MANAGEMENT] [-bpcd BOOTPREP_CONFIG_DIR] [-rv RECIPE_VARS]

> 	       [-sv SITE_VARS] [-mrs {reboot,stage}] [-cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE]

Reviewer 2: This line starts with a hard tab, change it to 8 spaces

  -cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE, --concurrent-management-rollout-percentage CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE
                        Limit the number of management nodes that roll out
                        concurrently based on the percentage specified. Must be an integer
  -smc, --set-management-config  
                        Specify the configuration name to be used for rebuilding the management node. If this parameter is provided, 
                        an image ID must also be specified using '--set-management-image'.

>                         Defaults to None, in which case the configuration stored for the activity during prepare-images stage

Reviewer: Small suggestion:
```suggestion
                        Defaults to None, in which case the configuration created during the prepare-images stage of the activity
```

			will be used.  
  -smi, --set-management-image  
                        Specify the image ID to be used for rebuilding the management node. If this parameter is provided,
                        a configuration name must also be specified using '--set-management-config'. 

>                         Defaults to None, in which case the image stored for the activity during prepare-images stage

Reviewer: ```suggestion
                        Defaults to None, in which case the image created during prepare-images stage of the activity
```

  -cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE, --concurrent-management-rollout-percentage CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE
                        Limit the number of management nodes that roll out
                        concurrently based on the percentage specified. Must be an integer

>   -smc, --set-management-config  

Reviewer: I believe this line was removed by accident. 
```suggestion
between 1-100. Defaults to 20 (percent).
-smc, --set-management-config
```

Reviewer 2: Also remove the trailing whitespace on this line

>                         Specify the configuration name to be used for rebuilding the management node. If this parameter is provided, 

Reviewer 2: Remove the trailing whitespace

                        an image ID must also be specified using '--set-management-image'.
                        Defaults to None, in which case the configuration created during prepare-images stage of the activity

> 			will be used.  

Reviewer 2: Replace the leading hard tabs with spaces, and remove the trailing whitespace

  -smi, --set-management-image  
                        Specify the image ID to be used for rebuilding the management node. If this parameter is provided,

>                         a configuration name must also be specified using '--set-management-config'. 

Reviewer 2: Remove trailing whitespace from 406 & 408

                        Defaults to None, in which case the image created during prepare-images stage of the activity

> 			will be used.  

Reviewer 2: Replace the leading hard tabs with spaces, and remove the trailing whitespace
